packages/teleport/src/cluster/components/Support/Support.tsx

```diff
@@ -0,0 +1,199 @@
+            )}
+            <SupportLink
+              title="Ask the Community Questions"
+              url="https://community.gravitational.com/"
+            />
+            <SupportLink
+              title="Request a New Feature"
+              url="https://github.com/gravitational/teleport/issues/new/choose"
+            />
+            <SupportLink
+              title="Send Product Feedback"
+              url="mailto:teleport-feedback@gravitational.com"
+            />
+          </Box>
+          <Box>
+            <Header title="Resources" icon={<Icons.Stars />} />
+            <SupportLink title="Quickstart Guide" url={docs.quickstart} />
+            <SupportLink title="tsh User Guide" url={docs.userManual} />
+            <SupportLink title="Admin Guide" url={docs.adminGuide} />
+            <SupportLink
+              title="Download Page"
+              url={
+                cfg.isEnterprise
+                  ? 'https://dashboard.gravitational.com/web/downloads '
+                  : 'https://gravitational.com/teleport/download'
+              }
+            />
+            <SupportLink title="FAQ" url={docs.faq} />
+          </Box>
+          <Box>
+            <Header title="Troubleshooting" icon={<Icons.Stars />} />
+            <SupportLink
+              title="Monitoring Teleport"
+              url={docs.troubleshooting}
+            />
+            <SupportLink
+              title="Collecting Debug Data"
+              url={docs.troubleshooting}
+            />
+            <SupportLink
+              title="Troubleshooting FAQ"
+              url={docs.troubleshooting}
+            />
+          </Box>
+          <Box>
+            <Header title="Updates" icon={<Icons.Stars />} />
+            <SupportLink
+              title="Product Changelog"
+              url="https://github.com/gravitational/teleport/blob/master/CHANGELOG.md"
+            />
+            <SupportLink
+              title="Gravitational Blog"
+              url="https://gravitational.com/blog/"
+            />
+          </Box>
+        </Flex>
+      </Card>
+      <Box
+        border="1px solid"
+        borderColor="primary.light"
+        mt={8}
+        mb={10}
+        borderRadius={3}
+        p={5}
+      >
+        <Text as="h5" typography="h5" mb={4} fontWeight="bold" caps>
+          Cluster Information
+        </Text>
+        <ClusterData title="Cluster Name" data={cluster.clusterId} />
+        <ClusterData title="Teleport Version" data={cluster.authVersion} />
+        <ClusterData title="Public Address" data={cluster.publicURL} />
+        <ClusterData title="Connected Nodes" data={cluster.nodeCount} />
+      </Box>
+    </FeatureBox>
+  );
+};
+
+/**
+ * getDocURLs returns an object of URL's appended with
+ * UTM, version, and type of teleport.
+ *
+ * @param version teleport version retrieved from cluster info.
+ */
+const getDocURLs = (version = '') => {
+  const verPrefix = cfg.isEnterprise ? 'e' : 'oss';
```

Comment:
Based on the spec we are missing `_`. Should be lke `oss_4.2.8`

Reply:
revised: https://github.com/gravitational/webapps/pull/78/commits/8cfd5ab1cdc5aa099a848bcd4e26bd26d30a1708